### ISSUE TITLE
Correct PKGBUILD for AUR release

### ISSUE THREAD

**Issue by morrieinmaas** [animo/siera#243]
The PKGBUILD for the AUR release should not use `sudo` commands. 

Currently the PKGBUILD used for AUR works but uses sudo commands. This is "not allowed" and subpar behaviour.

OUTCOME:
The PKGBUILD for the AUR has no sudo and works as expected. 

[For reference see eg this PKGBUILD](https://github.com/archlinux/svntogit-community/blob/packages/jless/trunk/PKGBUILD)

**Comment by blu3beri** [comment]
Just throwing it out here, very unaware of the details, but we might have to do something with FAKEROOT for sudo privileges.

**Comment by morrieinmaas** [comment]
yeah i think it's 'bad' practice because it'snot using the correct user space and also because it gives the wrong owner/privileges to what's installed. also I think the package manager doesn't remove the package correctly when added this way (but then who would want to remove siera anyway;))

**Comment by blu3beri** [comment]
@morrieinmaas what is the state of this issue? Is this something we should pick up or is the current working fine?

**Comment by iamwacko** [comment]
I wrote a fix for this. I noticed your package was orphaned (maybe because the maintainer heading was misspelled?), so I did something about it.
